utils/CMSApi/remotePublish.ts

```diff
@@ -0,0 +1,46 @@
+import { httpClient, StatusCode } from "configs/api/httpClient/HttpClient";
+import { CMS_API_JWT, CMS_API_VERSION, JWT, STATUS_DRAFT, STATUS_PUBLISH } from "constants/general";
+import { IError } from "interfaces";
+
+const updateJwt = async () => {
+  const result:{data:{token: string}} = await httpClient.post(`${process.env.NEXT_PUBLIC_BASE_URL}${process.env.NEXT_PUBLIC_BASE_ROUTE}${CMS_API_JWT}`, {
+    username: `${process.env.NEXT_PUBLIC_CMS_USERNAME}`,
+    password: `${process.env.NEXT_PUBLIC_CMS_PASSWORD}`
+  });
+    
+  if (result.data.token){
+    localStorage.setItem(JWT, result.data.token);
+  }
+};
+
+const createPost = async (CMS_URL:string, body: object, isDraft:boolean | undefined) => {
```

Comment:
```suggestion
const createPost = async (CMS_URL: string, body: object, isDraft:boolean | undefined) => {
```

I guess we need some typescript eslint plugin or maybe couple TS rules